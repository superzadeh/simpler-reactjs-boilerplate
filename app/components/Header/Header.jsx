import React from 'react';

class Header extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    return (
      <div className="header">
        Header
      </div>
    );
  }
}

export default Header;
