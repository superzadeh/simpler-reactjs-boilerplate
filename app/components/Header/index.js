import Header from './header';
import './style.scss';

export default Header;
