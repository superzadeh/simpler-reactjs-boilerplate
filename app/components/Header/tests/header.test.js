import React from 'react';
import Header from '../header';

describe('Header', () => {
  it('should render correctly', () => {
    const header = (<Header></Header>);
    expect(header).toMatchSnapshot();
  });
});
