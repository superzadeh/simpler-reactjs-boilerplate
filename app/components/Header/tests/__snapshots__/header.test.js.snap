exports[`Header should render correctly 1`] = `<Header />`;
